src/lib/prepareOas.ts

```diff
@@ -65,9 +65,18 @@ export default async function prepareOas(path: string, command: 'openapi' | 'ope
           const oas = new OASNormalize(file, { enablePaths: true });
           return oas
             .version()
-            .then(version => {
-              debug(`OpenAPI/Swagger version for ${file}: ${version}`);
-              return version ? file : '';
+            .then(({ specification, version }) => {
+              if (specification === 'openapi') {
+                debug(`OpenAPI version for ${file}: ${version}`);
+              } else if (specification === 'swagger') {
+                debug(`Swagger version for ${file}: ${version}`);
+              } else if (specification === 'postman') {
+                debug(`Postman version for ${file}: ${version}`);
+              } else {
+                return '';
+              }
+
+              return file;
             })
             .catch(e => {
               debug(`error extracting OpenAPI/Swagger version for ${file}: ${e.message}`);
```

Comment:
Maybe this debug should be more generic now?

```suggestion
              debug(`error extracting API definition specification version for ${file}: ${e.message}`);
```

Reply:
agree!